### ISSUE TITLE
Turn warns into errors 

### ISSUE THREAD

**Issue by rauchg** [zeit/eslint-config-node#3]
As discussed with @nkzawa, we should not be creating workflows where it's ok to push code with lingering warnings for others

**Comment by Qix-** [comment]
Was just thinking about this today. Agreed, but could you open the issue on `eslint-config-base`?

**Comment by matheuss** [comment]
Agreed. For example, `X is assigned but never used` should not even allow one to commit, IMHO 🙏
